src/shared/layout/sidebar/sidebar.tsx

```diff
@@ -13,19 +13,24 @@ import ListItemText from '@mui/material/ListItemText'
 import InboxIcon from '@mui/icons-material/MoveToInbox'
 import MailIcon from '@mui/icons-material/Mail'
 import AccountBalanceIcon from '@mui/icons-material/AccountBalance'
-import AttachMoneyIcon from '@mui/icons-material/AttachMoney';
+import AttachMoneyIcon from '@mui/icons-material/AttachMoney'
+import { useNavigate, NavLink } from 'react-router-dom'
 
-const drawerWidth = 240;
+const drawerWidth = 240
 
+type Props = {
+  user: number
+}
 
+export default function Sidebar({ user = 1 }: Props) {
+  const navigate = useNavigate()
 
-export default function Sidebar(props: any) {
   const MenuTargets = [
-    { title: 'Transactions', route: `/${props.user}/transactions` },
-    { title: 'Budgets', route: `/${props.user}/budgets` },
-    { title: 'Saving Plans', route: `/${props.user}/saving` },
-    { title: 'Faq', route: `/${props.user}/faq` },
-    { title: 'About us', route: `/${props.user}/omoss` },
```

Comment:
För att man ska kunna navigera till dessa som inloggad så ligger ${props.user}/ här för att kunna känna av vem som är inloggad och vems info som ska läsas in samt kolla om någon alls är inloggad eller om man ska rediectas till logIn. Det behöver vara kvar.